l10n_de_location_nuts/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2015 Tecnativa - Antonio Espinosa
+# Copyright 2015 Tecnativa - Jairo Llopis
+# Copyright 2015 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'NUTS Regions for German',
+    'summary': 'NUTS specific options for German',
+    'version': '12.0.1.0.0',
+    'category': 'Localisation/Europe',
+    'website': 'http://www.tecnativa.com',
```

Comment:
please change website to: https://github.com/OCA/l10n-germany

Reply:
Thanks again ;)